### PR TITLE
Caldavzap fixes

### PR DESCRIPTION
**Pull request by sg2342**

### CODE REVIEW

**Comment by hannesm** [comment]
I also set the content-type in https://github.com/roburio/caldav/commit/1f67b3a0ffb034025be8beb35858677d28435db9 (but your patch is smaller :) -- which browser(s) did you test?

File: README.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ diff -ur caldavzap/config.js mirage/caldavzap/config.js
 @@ -335,12 +335,12 @@
```
Comment (hannesm) [inline README.md]:
```suggestion
@@ -335,12 +335,11 @@
```

since below removes a line..

**Comment by hannesm** [comment]
and maybe we should directly advertise https://github.com/sg2342/caldavzap/ (or a fork thereof)? instead of having a diff in the README.

**Comment by hannesm** [comment]
I adjusted the README in the future branch, and also added 9cd4eef which may be of interest to you (always redirect `http://`, not serving the static files over http)

**Comment by sg2342** [comment]
> I also set the content-type in [1f67b3a](https://github.com/roburio/caldav/commit/1f67b3a0ffb034025be8beb35858677d28435db9) (but your patch is smaller :) -- which browser(s) did you test?

did not see this one, sorry. I tested chrome 73 and firefox 66.

**Pull request merged**